apps/bfd-model/pom.xml

```diff
@@ -49,6 +49,24 @@
 				<version>9.4.1208</version>
 				<scope>test</scope>
 			</dependency>
+
+			<dependency>
+				<!-- Used in subsitute of Java XML Bind -->
+				<groupId>jakarta.xml.bind</groupId>
+				<artifactId>jakarta.xml.bind-api</artifactId>
+				<version>2.3.2</version>
+			</dependency>
+			<dependency>
+				<groupId>org.glassfish.jaxb</groupId>
+				<artifactId>jaxb-runtime</artifactId>
+				<version>2.3.1</version>
+			</dependency>
+			<dependency>
+				<!-- Used for annotations in the generated code. -->
+				<groupId>jakarta.annotation</groupId>
+				<artifactId>jakarta.annotation-api</artifactId>
+				<version>1.3.5</version>
+			</dependency>
```

Comment:
I see some or all of these repeated. I think they should be pulled all the way up into app/pom.xml so that we don't run the risk of using two versions for any of them

Reply:
We can move these into app’s parent pom, but things like junit are in the parent poms of individual folders like model and server. So that begs the question if other dependencies in the repo could conform to a higher degree of dependencyManagement

Reply:
possibly, but I think we should keep the scope on this PR to just the things that changed.